Prox/Prox/Widgets/ReviewScoreView.swift

```diff
@@ -62,34 +57,58 @@ class ReviewScoreView: UIStackView {
         }
     }
 
+    private func updateReviewScore(score: Float) {
+        for (index, subview) in arrangedSubviews.enumerated() {
+            if let scoreItem = subview as? ReviewScoreItemView {
+                let scoreLeft = self.score - Float(index)
```

Comment:
nit: -> `scoreRemaining` or `scoreLeftOver`

I kept thinking left as in left/right.